### ISSUE TITLE
Language change

### ISSUE THREAD

**Issue by Minishlink** [didomi/react#86]
Hello,

How to reload the notice translations after a language change?
I tried setting the `config` prop that changes on locale change, and also calling `didomi.setConfigParameter('languages', { enabled: [locale], default: locale });` on locale change, but when showing the notice afterwards it is still in the previous language.

Thanks
Louis

**Comment by pierrevano** [comment]
Hello, the language of the notice cannot be changed after init. For the language to be applied, it needs to be set in the config as part of the initialization of the SDK.

**Comment by Minishlink** [comment]
Is the feature planned? On iOS/Android this is possible

**Comment by pierrevano** [comment]
No, it is not planned yet but I have forwarded your request to our product team.

**Comment by Kunzu2104** [comment]
Any update on this after 7 months ?

**Comment by pierrevano** [comment]
Hello, thank you for your comment. This is not yet prioritized, but it has been logged on our side. Please reach out to Didomi support for proper follow-up.

**Comment by Scipionh** [comment]
Hello,
Have you got any news on this matter? It is a massive hurdle when using Didomi with any SPA apps (React, Angular, ...)

**Comment by pierrevano** [comment]
Hello, please contact the Didomi support at support@didomi.io if you need visibility on this request. Best regards
